lib/dynflow/web/console.rb

```diff
@@ -37,6 +37,16 @@ class Console < Sinatra::Base
         erb :worlds
       end
 
+      post('/worlds/execution_items') do
```

Comment:
Again: Since we added more things than just the execution items. something more generic, such as execution_status or something like that would match more what it actually does.